Description: updating CMakeLists
 updating CMakeLists to v3 at least
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,11 @@
+cmake_minimum_required(VERSION 3.13.4)
+
 PROJECT(iDSK)
 
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
 add_executable(iDSK
 	src/Basic.cpp
 	src/BitmapCPC.cpp
